packages/back-end/src/api/experiments/getExperiment.ts

```diff
@@ -11,9 +18,40 @@ export const getExperiment = createApiRequestHandler(getExperimentValidator)(
       throw new Error("Could not find experiment with that id");
     }
 
+    const settings = req.context.org.settings;
+    const healthSettings = getHealthSettings(
+      settings,
+      orgHasPremiumFeature(req.context.org, "decision-framework")
+    );
+    let decisionCriteria = getDefaultDecisionCriteriaForOrg(settings);
+    if (settings?.defaultDecisionCriteriaId) {
+      try {
+        decisionCriteria ||=
+          (await req.context.models.decisionCriteria.getById(
+            settings!.defaultDecisionCriteriaId!
+          )) ?? DEFAULT_DECISION_CRITERIA;
+      } catch {
+        // Empty catch
+      }
+    }
```

Comment:
suggestion: We need to find better names for these.

We get the default decision criteria for an org, but it can return `undefined` if `settings.defaultDecisionCriteriaId` is not one of the hardcoded `DEFAULT_DECISION_CRITERIA(s)`; and if we get undefined back but we have a `settings.defaultDecisionCriteriaId` set then we fetch a `decisionCriteria` by `default criteria id`.

I am finding it confusing that we use `default` in 2 contexts here, one meaning the ones we (GB) provide to users out of the box, and two being the ones they set as default for their org/experiments.

Maybe we change the `DEFAULT_DECISION_CRITERIA` to be `PRESET_DECISION_CRITERIA` which would change the first function to be `getPresetDecisionCriteriaForOrg`?

cc @lukesonnet

Reply:
Yes, I'll refactor this in my banner PR.